File: 242-valid-anagram/valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
       vector<int> hashmap1 (26,0);
       vector<int> hashmap2 (26,0);
       if( s.size()!= t.size()) return false;
       for(int i = 0 ; i < s.size();i++){
         hashmap1[s[i]-'a']++;
         hashmap2[t[i]-'a']++;
       }
        return (hashmap1 == hashmap2);
    }
};

File: 242-valid-anagram/valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s) != len(t):
            return False
        
        hashmap1 = [0] * 26
        hashmap2 = [0] * 26
        
        for char in s:
            hashmap1[ord(char) - ord('a')] += 1
        
        for char in t:
            hashmap2[ord(char) - ord('a')] += 1
        
        return hashmap1 == hashmap2
